src/_locales/fr.config

```diff
@@ -120,7 +120,7 @@ Filtre+
 Statique
 
 @engine_dynamic
-Dynamiq.
+Dynamique
```

Comment:
This word is too long for the small button. In future versions "Filter" and "Filter+" will be merged, so there would be more space.